mycroft/skills/nest/__init__.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2016 Mycroft AI, Inc.
+#
+# This file is part of Mycroft Core.
+#
+# Mycroft Core is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Mycroft Core is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Mycroft Core.  If not, see <http://www.gnu.org/licenses/>.
+
+
+from os.path import dirname
+import subprocess
+
+from adapt.intent import IntentBuilder
+from mycroft.skills.core import MycroftSkill
+from mycroft.util.log import getLogger
+
+__author__ = 'crios'
+
+LOGGER = getLogger(__name__)
+
+
+class NestSkill(MycroftSkill):
+    def __init__(self):
+        super(NestSkill, self).__init__(name="NestSkill")
+        self.nest_user = self.config.get('nest_user')
+        self.nest_password = self.config.get('nest_password')
```

Comment:
I will discuss this with them to ensure this is a good decision, I do not want to confuse Cerberus.